File: src/main/java/bot/commands/SubjectsCommand.java
package bot.commands;

import bot.run.AbitVTBot;
import org.telegram.telegrambots.meta.api.objects.Update;
import org.telegram.telegrambots.meta.exceptions.TelegramApiException;

import java.util.ResourceBundle;

public class SubjectsCommand implements Command {

    private final AbitVTBot bot;

    public SubjectsCommand(AbitVTBot bot) {
        this.bot = bot;
    }

    @Override
    public void execute(Update update) throws TelegramApiException {

    }

    @Override
    public String help(String chatId) {
        ResourceBundle helpBundle = ResourceBundle.getBundle("help", bot.getSql().selectLanguage(chatId).getLocale());
        return helpBundle.getString("subjects");
    }

    @Override
    public String name() {
        return "/subjects";
    }

}

File: src/main/java/bot/commands/SubjectsCommand.kt
package bot.commands

import bot.run.AbitVTBot
import org.telegram.telegrambots.meta.api.objects.Update

class SubjectsCommand(private val bot : AbitVTBot) : Command {
    override fun execute(update: Update) {
    }

    override fun help(chatId: String): String {
        return bot.resourceOperator.getHelp("subjects", chatId)!!
    }

    override fun name(): String {
        return "/subjects"
    }
}